src/core/utils/components/Popover.tsx

```diff
@@ -38,7 +38,7 @@ export type PopoverProps = {
 export const Popover = React.forwardRef((props: PopoverProps, ref) => {
   const [mounted, setMounted] = React.useState(false);
 
-  const tippyRef = React.useRef(null);
+  const tippyRef = React.useRef<Element>(null);
```

Comment:
Why did we avoid specifying the type before? 🤔

Reply:
I dont know, less writing?

Reply:
no I think it caused an error somewhere. Can't remember now 🤷‍♀️